src/official-widgets/merchandise-search-bar/default-config.ts

```diff
@@ -0,0 +1,387 @@
+    suggestions: '提案',
+    viewAllProducts: 'すべての製品を見る',
+    popularChoices: '人気の選択',
+    trending: 'トレンド',
+    errorMessage: '無効なクエリまたは画像を入力しました。もう一度お試しください。',
+  },
+  th: {
+    searchBarPlaceholder: 'คุณกำลังมองหาอะไร?',
+    uploadScreenTitle: 'แสดงให้เราเห็นสิ่งที่คุณกำลังมองหา',
+    dragImageToSearch: 'ลากรูปภาพเพื่อค้นหาหรือคลิกเพื่อเรียกดู',
+    tapToSearchImage: 'แตะที่นี่เพื่อค้นหารูปภาพ',
+    tapProductGallery: 'หรือแตะแกลเลอรีผลิตภัณฑ์ยอดนิยมของเราด้านล่าง',
+    relatedProducts: 'สินค้าที่เกี่ยวข้อง',
+    recentSearches: 'การค้นหาล่าสุด',
+    suggestions: 'คำแนะนำ',
+    viewAllProducts: 'ดูสินค้าทั้งหมด',
+    popularChoices: 'ตัวเลือกยอดนิยม',
+    trending: 'กำลังมาแรง',
+    errorMessage: 'คุณได้ป้อนคำค้นหาหรือรูปภาพที่ไม่ถูกต้อง โปรดลองอีกครั้ง',
+  },
+};
+
+export const DEFAULT_CUSTOMIZATIONS: WidgetConfig['customizations'] = {
+  breakpoints: {
+    mobile: {
+      maxWidth: 767,
+    },
+    tablet: {
+      maxWidth: 1023,
+    },
+  },
+  productGrid: {
+    mobile: {
+      productsPerRow: 2,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+    tablet: {
+      productsPerRow: 3,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+    desktop: {
+      productsPerRow: 4,
+      marginVertical: 16,
+      marginHorizontal: 8,
+    },
+  },
+  productCard: {
+    imageAspectRatio: '2 / 3',
+    openLinksInNewTab: false,
+    price: {
+      show: true,
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+      fontColor: '#EF4444',
+      fontColorDark: '#EF4444',
+    },
+    originalPrice: {
+      show: true,
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+      fontColor: '#9CA3AF',
+      fontColorDark: '#9CA3AF',
+    },
+    title: {
+      show: true,
+      fieldSource: 'title',
+      font: {
+        mobile: {
+          size: 14,
+          weight: 700,
+        },
+        tablet: {
+          size: 14,
+          weight: 700,
+        },
+        desktop: {
+          size: 16,
+          weight: 700,
+        },
+      },
+    },
+    secondaryTitle: {
+      show: false,
+      fieldSource: 'brand',
+      font: {
+        mobile: {
+          size: 12,
+          weight: 400,
+        },
+        tablet: {
+          size: 12,
+          weight: 400,
+        },
+        desktop: {
+          size: 14,
+          weight: 400,
+        },
+      },
+    },
+  },
+  imageUpload: {
+    enable: true,
+    icon: {
+      color: '#929292',
+      colorDark: '#929292',
+    },
+    images: [
+      {
+        url: 'https://cdn.visenze.com/images/widget-1.jpg',
```

Comment:
Should be updated with the recent images; refer to https://github.com/visenze/visenze-experiences-web/pull/80.